packages/build/src/status/validations.ts

```diff
@@ -0,0 +1,19 @@
+import { SecretScanResult } from '../plugins_core/secrets_scanning/utils.js'
+
+// Reports any validations completed on the deploy to the API
+export const reportValidations = async function ({
+  api,
+  secretScanResult,
+  deployId,
+}: {
+  api: unknown
+  secretScanResult: SecretScanResult
+  deployId: string
+}) {
+  try {
+    // @ts-expect-error API type is not defined
```

Comment:
Ideally we aren't adding in a new `@ts-expect-error` unless we absolutely need to

Reply:
I'll see what I can do. It doesn't look like `NetlifyAPI` contains the type definitions for the dynamically created functions like `createPluginRun` or this `updateDeployValidations`

Reply:
Why don't we want to bundle them into the type definitions? I'm not saying we do want to, I just want to understand